core/src/main/java/com/google/googlejavaformat/java/JavaCommentsHelper.java

```diff
@@ -112,7 +112,15 @@ private String indentLineComments(List<String> lines, int column0) {
       Matcher matcher = LINE_COMMENT_PREFIX.matcher(line);
       if (matcher.find()) {
         int length = matcher.group(1).length();
-        line = Strings.repeat("/", length) + " " + line.substring(length);
+        // If it is `//noinspection` then don't add a space.  `//noinspection` is
+        // used for single-line suppressions by both IntelliJ and Android Lint,
+        // and for now there is no good alternative.
+        boolean hasNoSuppression =
```

Comment:
This should either be `hasSuppression` or `hasNoInspection`. Currently the naming is inaccurate since it's true when there is a suppression.